src/style.css

```diff
@@ -0,0 +1,108 @@
+@import url("./stars.css");
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --black-color: #060b35;
+  --gray-color: #616070;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
```

Comment:
The card width should be 200px according to the task description. Update the width to 200px.

Reply:
(you always can add `box-sizing:border-box;`)